lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,21 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        case app_name
+        when '.'
+          ::File.basename(Dir.getwd)
+        when '/'
```

Comment:
@jodosha should we allow path such as the one @joneslee85 has mentioned eg
```shell
lotus new lib/my_new_lotus_app
# would create my_new_lotus_app inside lib
```
then we can raise exception when app_name is '/'